pkg/bundle/files.go

```diff
@@ -4,42 +4,80 @@ import (
 	"fmt"
 	"os"
 	"path/filepath"
+	"sync"
 
 	"github.com/RedHatGov/bundle/pkg/config/v1alpha1"
+	"github.com/gammazero/workerpool"
+	"github.com/sirupsen/logrus"
 )
 
+type Reconciler struct {
```

Comment:
Typically I either define an interface for a type that requires instantiation, or permit the struct to be used directly with an `init()` method called in exported methods; in this case I'd go with the latter option. Right now `Reconciler` can be used directly but will panic since its unexported fields are unset. For example:
```go
type Reconciler struct {
	newFiles   map[string]v1alpha1.File
	foundFiles map[string]struct{}
	mu         *sync.Mutex
    initOnce sync.Once
}

func (r *Reconciler) init() {
	r.initOnce.Do(func() {
		r.newFiles = make(map[string]v1alpha1.File)
		r.foundFiles = make(map[string]struct{})
		r.mu = new(sync.Mutext)
	})
}

func (r *Reconciler) Files(v1alpha1.Metadata, string) (map[string]v1alpha1.File, error) {
	r.init()
	...
}
```

Reply:
Resolving. End up reworking this over the weekend and found that I didn't really need the struct.